lib/vanagon/project/dsl.rb

```diff
@@ -317,6 +317,10 @@ def package_override(var)
         platform = @project.platform
         platform.package_override(self._project, var)
       end
+
+      def only_fetch(path)
```

Comment:
Please add heredoc for the new method

Reply:
I'm also torn here because I find calling this `only_fetch` confusing (mostly, it sounds to me like something that's going to be a binary setting), but it's not clear to me what a better name might be. Might be better to have some boolean parameter to disable the extra make targets, along with more generalized functionality to customize which artifact(s) you want to retrieve?

Reply:
I was thinking the same thing re: confusing name.

Reply:
@underscorgan I had the exact same thoughts. I also thought about the two parameter approach and was going to wait and see what y'all thought of this approach first.

I'll update the PR with a second bool param